### ISSUE TITLE
Nested Data with Graphs (multilevel pie chart)

### ISSUE THREAD

**Issue by vforvasquez** [swimlane/ngx-charts#296]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**

We can make pie graphs with one level of data. 
Ex: 50% books, 50% furniture.
I would like to be able to make a pie graph with data that is broken down further.
Ex: 50% books->25% fiction, 25% non-fiction; 50% furniture -> 25% chairs, 25% tables.
I apologize if this is currently available, I haven't seen it mentioned anywhere.

**Expected behavior**

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **ngx-charts version:** x.x.x

* **Angular version:** 2.x.x

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by marjan-georgiev** [comment]
I'm not sure how that would look like on a pie chart. We certainly support it on other charts (like the grouped bar chart).

Can you show me an example of this chart?

**Comment by vforvasquez** [comment]
Sure, here is an example of what I'm talking about - obviously a lot more advanced though: 
https://i.stack.imgur.com/Yx83O.png

**Comment by marjan-georgiev** [comment]
Ah, right, thanks.

I like the idea. I think it should be implemented as a separate chart though, otherwise the pie chart would be too complicated.

**Comment by vforvasquez** [comment]
Cool, thanks. So if it is a whole new type of chart I'm guessing it would take a while to see anything, right?

**Comment by marjan-georgiev** [comment]
Yes. Unfortunately it is not just a simple modification we could make to the pie chart. If anyone is willing to take a shot at implementing it, I'd be happy to help guiding and reviewing.

**Comment by vforvasquez** [comment]
Awesome, thanks!

**Comment by vforvasquez** [comment]
Awesome, thanks!

**Comment by RajIET** [comment]
@marjan-georgiev Please give be brief description how you handle the single level pie chart. I want to work on it.
Please share working guidelines on the project.
you can also share me on raj1.aggrawal@gmail.com
Thanks in Advance

**Comment by vishals82** [comment]
@RajIET: any progress on this chart?

**Comment by Alkasih** [comment]
is this bug fixed?

**Comment by faizan78693** [comment]
Is it possible on treeMap ?
We have interactive tree map when we have children inside an Object, but how can i show whole data in one view , parent should contain children, inside of them.

Something like we have here :- 
https://observablehq.com/@d3/treemap

**Comment by KingDarBoja** [comment]
Willing to implement this multi level pie chart, just need some free time as I did with my previous contribution.

**This is a friendly reminder for myself.**

**Comment by Volftsun** [comment]
any updates?
